Source/Scene/GlobeDepth.js

```diff
@@ -0,0 +1,235 @@
+            destroyAttachments : false
+        });
+
+        var complete = WebGLRenderingContext.FRAMEBUFFER_COMPLETE;
+        if (globeDepth.framebuffer.status !== complete || globeDepth._copyDepthFramebuffer.status !== complete) {
+            destroyTextures(globeDepth);
+            destroyFramebuffers(globeDepth);
+            return false;
+        }
+
+        return true;
+    }
+
+    function updateFramebuffers(globeDepth, context) {
+        if (!context.depthTexture) {
+            return;
+        }
+
+        var width = context.drawingBufferWidth;
+        var height = context.drawingBufferHeight;
+
+        var colorTexture = globeDepth._colorTexture;
+        var textureChanged = !defined(colorTexture) || colorTexture.width !== width || colorTexture.height !== height;
+        if (!defined(globeDepth.framebuffer) || textureChanged) {
+            if (!createFramebuffers(globeDepth, context, width, height)) {
+                // framebuffer creation failed
+                return false;
+            }
+        }
+
+        context.uniformState.globeDepthTexture = globeDepth._depthStencilGlobeTest;
+
+        return true;
+    }
+
+    function updateCopyCommands(globeDepth, context) {
+        if (!defined(globeDepth._copyDepthCommand)) {
+            globeDepth._copyDepthCommand = context.createViewportQuadCommand(PassThrough, {
+                renderState : context.createRenderState(),
+                uniformMap : {
+                    u_texture : function() {
+                        return globeDepth._depthStencilTexture;
+                    }
+                },
+                owner : globeDepth
+            });
+        }
+
+        globeDepth._copyDepthCommand.framebuffer = globeDepth._copyDepthFramebuffer;
+
+        if (!defined(globeDepth._copyColorCommand)) {
+            globeDepth._copyColorCommand = context.createViewportQuadCommand(PassThrough, {
+                renderState : context.createRenderState(),
+                uniformMap : {
+                    u_texture : function() {
+                        return globeDepth._colorTexture;
+                    }
+                },
+                owner : globeDepth
+            });
+        }
+
+        if (!defined(globeDepth._clearColorCommand)) {
+            globeDepth._clearColorCommand = new ClearCommand({
+                color : new Color(0.0, 0.0, 0.0, 0.0),
+                owner : globeDepth
+            });
+        }
+
+        globeDepth._clearColorCommand.framebuffer = globeDepth.framebuffer;
+    }
+
+    GlobeDepth.prototype.update = function(context) {
+        if (!this.supported) {
+            return;
+        }
+
+        updateFramebuffers(this, context);
+        updateCopyCommands(this, context);
+    };
+
+    GlobeDepth.prototype.getFramebuffer = function() {
+        return this._frambuffer;
+    };
+
+    GlobeDepth.prototype.executeCopyDepth = function(context, passState) {
```

Comment:
Ah, I see the symmetry with `executeCommand`...not sure that is a good reason though.  Whatever you think.